configuration.go

```diff
@@ -0,0 +1,85 @@
+package caddy
+
+import (
+	"github.com/darkweak/souin/configurationtypes"
+	"go.uber.org/zap"
+)
+
+// DefaultCache the struct
+type DefaultCache struct {
+	Distributed bool
+	Headers     []string
+	Olric       configurationtypes.CacheProvider
+	Regex       configurationtypes.Regex
+	TTL         string
```

Comment:
This should be a `time.Duration`

Reply:
Related to the core: https://github.com/Darkweak/Souin/issues/94